app/views/shared/_header.html.haml

```diff
@@ -32,7 +32,7 @@
               %li.visible-with-nav= link_to t('pages.cart') + " (#{current_cart.order_details.count})", :cart
               %li.divider-vertical
               = render "shared/message_summary"
-              %li= link_to t('pages.logout'), sign_out_user_path
+              %li= render "shared/logout/link_#{warden.session(:user)[:strategy] || 'default'}"
```

Comment:
This seems like a place where I might actually prefer a view helper rather than having this or statement directly in the view.

Reply:
View helper is a good call.